### PR TITLE
docs: manually verify case signature

### PR DESCRIPTION
**Pull request by jeyaramashok**
Changes:

- Add instructions and script to manually verify CASE signature

### CODE REVIEW

**Comment by m-g-k** [comment]
Hi @jeyaramashok I just took a look at this and I followed your instructions and it does verify that the signature inside the case .tgz is correct. 

However, and I may be missing something, it does not verify that the case file is actually "intact". For example if I edit the case file to change it's contents, as long as I don't change the signature file it seems to verify as OK. 

To me this is misleading for customers who may assume verifying the signature would proves that the case file has not been tampered with which does not appear to be the case. 

Is the plan to extend your work here to perform full image verification? If not, I think the docs need to be much clearer as to what is actually being verified. 

Many thanks, MGK

**Comment by jeyaramashok** [comment]
@m-g-k thanks for catching this, just noticed there is a bug in the script, let me fix it now

**Comment by jeyaramashok** [comment]
@m-g-k I cleaned the README now. I got confused earlier, the purpose of this script is to just validate the signature of downloaded CASE and check if it's signed by IBM ( authenticity ). The full rigorous check including detecting any modifications & signature can be done only by ibm-pak.

File: docs/verify-case-signature.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+- [yq](https://mikefarah.gitbook.io/yq/)
+- [jq](https://stedolan.github.io/jq/)
+- [openssl](https://www.openssl.org/)
+- [cosign](https://github.com/sigstore/cosign/releases) (optional)
+
+
+## Download public key and certificate
+
+```bash
+$ mkdir pub_keys
```
Comment (Man1902) [inline docs/verify-case-signature.md:33]: Looks like `cd pub_keys` command needs to be added prior to downloading keys as the same path is getting referred in verification command.

File: docs/verify-case-signature.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+-rw-r--r--  1 jey  staff   316K Mar  2 01:36 /Users/jey/.ibm-pak/data/cases/ibm-cp-common-services/1.19.1/ibm-cs-monitoring-1.19.1.tgz
+-rw-r--r--  1 jey  staff    61K Mar  2 01:36 /Users/jey/.ibm-pak/data/cases/ibm-cp-common-services/1.19.1/ibm-platform-api-operator-1.19.1.tgz
+```
+
+## Verify
+
+### Using OpenSSL
+
+```bash
+$ ./verify-case-signature.sh --case ./cases/ibm-cp-common-services-1.19.1.tgz --key ./pub_keys/ibm-pak.pem.pub.key --cert ./pub_keys/ibm-pak.pem.cer --cert-chain ./pub_keys/ibm-pak.pem.chain
```
Comment (Man1902) [inline docs/verify-case-signature.md:105]: As per the above case directory structure, looks like in `--case` args path should be suggested as  `/Users/jey/.ibm-pak/data/cases/ibm-cp-common-services/1.19.1/ibm-cp-common-services-1.19.1.tgz`

Comment (Man1902) [inline docs/verify-case-signature.md:105]: While verifying case signature , getting error for both using OpenSSL as well as cosign
```
root@bastion-airgap-e2e-akt-airgap-test ~]# ./verify-case-signature.sh --case /root/.ibm-pak/data/cases/ibm-cp-common-services/1.19.1/ibm-cp-common-services-1.19.1.tgz --key ./pub_keys/ibm-pak.pem.pub.key --cert ./pub_keys/ibm-pak.pem.cer --cert-chain ./pub_keys/ibm-pak.pem.chain
Error: open /tmp/casedir.OP70kHw8zc/signature.yaml: no such file or directory

[root@bastion-airgap-e2e-akt-airgap-test ~]# ./verify-case-signature.sh --case /root/.ibm-pak/data/cases/ibm-cp-common-services/1.19.1/ibm-cp-common-services-1.19.1.tgz --key ./pub_keys/ibm-pak.pem.pub.key --cert ./pub_keys/ibm-pak.pem.cer --cert-chain ./pub_keys/ibm-pak.pem.chain --tool cosign
Error: open /tmp/casedir.FM3G7JyGuh/signature.yaml: no such file or directory

[root@bastion-airgap-e2e-akt-airgap-test ~]# yq --version
yq (https://github.com/mikefarah/yq/) version v4.31.2
```

Looks like there is change in command signature as per the yq version usage (https://mikefarah.gitbook.io/yq/#notice-for-v4.x-versions-prior-to-4.18.1). Please take a look.

**Comment by jeyaramashok** [comment]
We had a discussion and exploring few other options to do the complete verification. I will close this PR now and create a new one with the updated changes.

**Pull request closed without merging**